### DISCUSSION TITLE
Per-group visibility settings

### DISCUSSION THREAD

**Discussion by dmezh** [outline/outline#2614] [Ideas]
Currently, Outline only lets non-admins see groups that they are themselves members of (commit 43493fcbbdf89d38b37cb0eb82b49ef1027c193b).

Per-group visibility settings would be great, because then users could use groups to more effectively share private collections. A toggle that makes the group 'private' or 'public' would accomplish this.

**Comment by tommoor** [comment]
Hey @dmezh could you lay out a scenario where team visible groups would be useful to help understand? Thanks

**Reply by dmezh** [reply]
Hey @tommoor , one good example is many-to-few sharing. Say we have 15 people that all want to share individual collections or docs with a group of 3 people - this scenario came up for us:

- We're using Outline for documentation for a school extracurricular team
- Some students need to share their private notebooks (Outline collections) with a group of instructors or leads
- The only way to do this now is with share links or adding each person who needs access individually, because the students can't see the instructor groups.

**Reply by tommoor** [reply]
Thanks, this is a good example! I don't see any harm in adding this feature as long as existing groups are defaulted to hidden
